### PR TITLE
feat: prepare package for first release

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

- update Readme.md with most important infos
- add CJS and ESM transformation
- update package.json infos
- add `prepublishOnly` script
- add CODEOWNERS file

cc @imabp

### CODE REVIEW

**Comment by derberg** [comment]
maybe make it `feat` to trigger release after merge

File: README.md [hunk 1]
```diff
@@ -1,3 +1,63 @@
...
+## Develop
+
+1. Write code and tests in the `__tests__` folder.
+2. Make sure all tests pass by `npm test` command.
+3. Make sure code can be transformed to JS by `npm run build` command.
+4. Make sure code is well formatted and secure by `npm run lint:fix` command.
+
+## Contributing
+
+Read [CONTRIBUTING](https://github.com/asyncapi/.github/blob/master/CONTRIBUTING.md) guide.
```
Comment (derberg) [inline README.md:63]: you have it in this repo too, we now replicate it, so no need to link to `.github` repo

Comment (magicmatatjahu) [reply README.md]: Thanks for catch! :) Changed.

**Pull request merged**